jena-fuseki2/jena-fuseki-embedded/src/main/java/org/apache/jena/fuseki/embedded/FusekiServer.java

```diff
@@ -20,10 +20,9 @@
 
 import static java.util.Objects.requireNonNull;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Ditto first comment, about maintaining the current standard in the code (I think)